server/src/main/java/com/objectcomputing/checkins/services/frozen_template_questions/FrozenTemplateQuestionController.java

```diff
@@ -0,0 +1,105 @@
+package com.objectcomputing.checkins.services.frozen_template_questions;
+import io.micronaut.http.HttpResponse;
+import io.micronaut.http.annotation.*;
+import io.micronaut.scheduling.TaskExecutors;
+import io.micronaut.security.annotation.Secured;
+import io.micronaut.security.rules.SecurityRule;
+import io.netty.channel.EventLoopGroup;
+import io.reactivex.Single;
+import io.reactivex.schedulers.Schedulers;
+import io.swagger.v3.oas.annotations.tags.Tag;
+
+import javax.annotation.Nullable;
+import javax.inject.Named;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import java.net.URI;
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.ExecutorService;
+import java.util.stream.Collectors;
+
+
+@Controller("/services/feedback/frozen_template_questions")
```

Comment:
In our other services I think we have everywhere used hyphen(-) for controller path so to be consistent lets change it to use hyphen instead of underscore.

Reply:
Changed to be hyphens everywhere!